nucypher/network/resources.py

```diff
@@ -3,8 +3,9 @@
 import os
 from hendrix.facilities.resources import MediaResource
 
+
 def get_static_resources():
     resources = []
     if os.getenv('NUCYPHER_STATIC_FILES_ROOT'):
-        resources.append(MediaResource(os.getenv('NUCYPHER_STATIC_FILES_ROOT'), namespace='statics'))
+        resources.append(MediaResource(os.getenv('NUCYPHER_STATIC_FILES_ROOT').encode(), namespace=b'statics'))
```

Comment:
This was the twisted change that led to the hendrix change, that led to this line needing to be changed?

Reply:
It's related.  Twisted's `putChild` function expects bytes for paths as of 22.10.0.